src/main/java/org/openlmis/requisition/domain/requisition/StatusChangeValidationService.java

```diff
@@ -64,8 +64,13 @@ public StatusChangeValidationService(Requisition requisition, LocalDate currentD
     validators.add(new RequestedQuantityValidator(requisition, orderables));
     validators.add(new StockAdjustmentsValidator(requisition, orderables));
     validators.add(new TotalFieldValidator(requisition, requisition.getTemplate(), orderables));
-    validators.add(new TotalReceivedQuantityValidator(requisition,
-        requisition.getTemplate(), orderables));
+
+    boolean skipTotalReceivedQuantityValidation = Boolean.TRUE.equals(requisition.getTemplate()
+        .getPatientsTabEnabled()) && !requisition.getStatus().isAuthorized();
+    if (!skipTotalReceivedQuantityValidation) {
```

Comment:
I'm wondering if this is a nice approach - not adding a validator when PatientsTabEnabled=true.

TotalReceivedQuantityValidator takes requisition and checks if a Received Quantity inside this Requisition is valid. 
When it's valid? 
Before changes: When it's not null.
After changes: When it's not null or PatientsTabEnabled is true.

I think it would make cleaner code if the PatientsTabEnabled check would be inside the validator. I would move it, but it's up to you.

Reply:
I moved this logic to TotalReceivedQuantityValidator class